src/main/java/org/jitsi/jigasi/stats/StatsHandler.java

```diff
@@ -316,6 +311,24 @@ private void createAndStartCallPeriodicRunnable(
         int interval = accountID.getAccountPropertyInt(
             CS_ACC_PROP_STATISTICS_INTERVAL, DEFAULT_STAT_INTERVAL);
 
+        String conferenceName = callContext.getRoomName();
+        if (conferenceName.contains("@"))
+        {
+            conferenceName = Util.extractCallIdFromResource(conferenceName);
```

Comment:
maybe it would be worth moving this logic to a `getConferenceName` getter inside the `CallContext`?